test/runnable/test15371.d

```diff
@@ -0,0 +1,31 @@
+// EXTRA_SOURCES: imports/test15371.d
+
+/*
+TEST_OUTPUT:
+---
+
+---
+*/
+
+module test15371;
+
+import imports.test15371;
+import core.stdc.stdio;
+
+void main() {
+    auto test = new Test15371();
+    const auto hasMember = __traits(hasMember, test, "privateField");
+    if (hasMember) {
+        printf("_traits(hasMember) found privateField\r\n");
```

Comment:
I've never seen anyone use `\r\n` when printing to the console.

Reply:
Without it is just kept printing on the same line. And only with the failing assert.

Reply:
I mean, only `\n` is usually used.

Reply:
This works correctly on Windows too.

Reply:
You almost never need `\r` on Windows, too.

Reply:
It's an ingrained habit from the bad old days. I'll remove it.